apps/ejabberd/src/mongoose_privacy.erl

```diff
@@ -0,0 +1,66 @@
+%%%-------------------------------------------------------------------
+%%% @author bartek <bartlomiej.gorny@erlang-solutions.com>
+%%% @copyright (C) 2017, Erlang Solutions sp. z o.o.
+%%% @doc
+%%%
+%%% @end
+%%% Created : 06. Feb 2017 11:37
+%%%-------------------------------------------------------------------
+-module(mongoose_privacy).
+-author("bartek").
+
+-include("ejabberd.hrl").
+-include("mod_privacy.hrl").
+
+%% API
+-export([privacy_check_packet/6, privacy_check_packet/7]).
+
+%%% API %%%
+
+-spec privacy_check_packet(Acc :: mongoose_acc:t(),
```

Comment:
Ah, got it. I was confused because I saw line 20 "-spec privacy_check_packet(Acc :: mongoose_acc:t()," and Elvis complaining about use of a record...